CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to `data-transfer-object` will be documented in this file
 
 - Fix for an empty value being created when casting `ArrayAccess` objects (#216)
 - Add logic exception when attempting to cast `ArrayAccess` objects that are not traversable (#216)
+<<<<<<< HEAD
```

Comment:
Careful, there's a merge conflict committed here

Reply:
Yeah that was my bad (🤦🏻‍♂️), I caught it too late. Everything should be clean in the latest commit, so a squash and merge would clean up that history. Alternatively, I can clean up the Git history manually with a reset.

Reply:
Don't worry about the history :)

Let me know when this is ready?